cwms_radar_api/src/main/java/cwms/radar/ApiServlet.java

```diff
@@ -196,82 +202,113 @@ public void init() throws ServletException{
-        radarCrud("/ratings/spec/{rating-id}", new RatingSpecController(metrics), requiredRoles);
-        radarCrud("/ratings/{rating}", new RatingController(metrics), requiredRoles);
-        radarCrud("/catalog/{dataset}", new CatalogController(metrics), requiredRoles);
-        radarCrud("/basins/{basin-id}", new BasinController(metrics), requiredRoles);
-        radarCrud("/blobs/{blob-id}", new BlobController(metrics), requiredRoles);
-        radarCrud("/clobs/{clob-id}", new ClobController(metrics), requiredRoles);
-        radarCrud("/pools/{pool-id}", new PoolController(metrics), requiredRoles);
-        radarCrud("/specified-levels/{level-id}", new SpecifiedLevelController(metrics), requiredRoles);
+        radarCrud("/ratings/template/{template-id}",
+                new RatingTemplateController(metrics), requiredRoles);
+        radarCrud("/ratings/spec/{rating-id}",
+                new RatingSpecController(metrics), requiredRoles);
+        radarCrud("/ratings/{rating}",
+                new RatingController(metrics), requiredRoles);
+        radarCrud("/catalog/{dataset}",
+                new CatalogController(metrics), requiredRoles);
+        radarCrud("/basins/{basin-id}",
+                new BasinController(metrics), requiredRoles);
+        radarCrud("/blobs/{blob-id}",
+                new BlobController(metrics), requiredRoles);
+        radarCrud("/clobs/{clob-id}",
+                new ClobController(metrics), requiredRoles);
+        radarCrud("/pools/{pool-id}",
+                new PoolController(metrics), requiredRoles);
+        radarCrud("/specified-levels/{level-id}",
+                new SpecifiedLevelController(metrics), requiredRoles);
     }
 
 
     /**
      * This method is very similar to the ApiBuilder.crud method but the specified roles
-     * are only required for the post, patch and delete methods.    getOne and getAll are always allowed.
+     * are only required for the post, patch and delete methods.  getOne and getAll are always
+     * allowed.
      * @param path where to register the routes.
      * @param crudHandler the handler requests should be forwarded to.
-     * @param roles the accessmanager will require these roles are present to access post, patch and delete methods
+     * @param roles the accessmanager will require these roles are present to access post, patch
+     *             and delete methods
      */
-    public static void radarCrud(@NotNull String path, @NotNull CrudHandler crudHandler, @NotNull RouteRole... roles) {
+    public static void radarCrud(@NotNull String path, @NotNull CrudHandler crudHandler,
+                                 @NotNull RouteRole... roles) {
         String fullPath = prefixPath(path);
-        String[] subPaths = Arrays.stream(fullPath.split("/")).filter(it -> !it.isEmpty()).toArray(String[]::new);
+        String[] subPaths = Arrays.stream(fullPath.split("/"))
+                .filter(it -> !it.isEmpty()).toArray(String[]::new);
```

Comment:
not sure if we can codify this in checkstyle (and you don't need to change this) but I've always preferred the visual of the following:

```java 
...  = Array.stream(fullPath.split("/"))
                .filter(it -> !it.isEmpty())
                .toArray(String[]::new);
```

Always seems clearer to me if each operation is on it's own line.